src/rpc/mining.cpp

```diff
@@ -492,6 +492,7 @@ static RPCHelpMan getprioritisedtransactions()
                 {RPCResult::Type::OBJ, "<transactionid>", "", {
                     {RPCResult::Type::NUM, "fee_delta", "transaction fee delta in satoshis"},
                     {RPCResult::Type::BOOL, "in_mempool", "whether this transaction is currently in mempool"},
+                    {RPCResult::Type::NUM, "modified_fee", /*optional=*/true, "new fee for this transaction. Only returned if transaction is in mempool"},
```

Comment:
dc189d3ec5c4af9bca02928dfeb4770570060982: I'm not sure that "new fee" would be understood by people. It would also be helpful to add the units here, i.e.
```suggestion
                    {RPCResult::Type::NUM, "modified_fee", /*optional=*/true, "modified fee in satoshis. Only returned if in_mempool=true"},
```

Reply:
updated in cfdbcd19b32fd63954d7947dcc639aef291fb6b2